oauth2client/client.py

```diff
@@ -771,11 +771,11 @@ def _do_refresh_request(self, http):
         headers = self._generate_refresh_request_headers()
 
         logger.info('Refreshing access_token')
-        resp, content = transport.request(
+        response = transport._request(
```

Comment:
Comment for the entire commit where `transport._request` goes private:

Doesn't that break your rule of module-private for private-module?